pkg/rpc/server/server.go

```diff
@@ -78,6 +78,9 @@ type (
 	}
 )
 
+// ErrUnknownTransactionMessage is returned when transaction is not found.
+const ErrUnknownTransactionMessage = "Unknown transaction"
```

Comment:
So to use one needs to import whole rpc server into his project?

Reply:
Removed.